### DISCUSSION TITLE
image stream插件点击unsplash图库突然间出现问题

### DISCUSSION THREAD

**Discussion by IshuaWang** [halo-dev/halo#7124] [Q&A]
昨天还没出现，今天突然出现了这个错误，其他两个图库没有问题，并没有修改这个图库的配置。不知道是为什么？
![image](https://github.com/user-attachments/assets/60696472-0745-434a-95b5-0a69d151e19d)

**Comment by ruibaby** [comment]
https://github.com/halo-sigs/plugin-image-stream?tab=readme-ov-file#%E7%89%B9%E6%80%A7
